blackbox_repository/tabulated_benchmark.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
General question: I am just trying to learn here. I had this trouble to support the remote launcher, where I had to pass a class name and create the table object only when first used. You don't have any of these issues here at all.

Is this simply because blackbox_repository is part of the syne_tune code, while the code for the table (say, NASBench201Benchmark) is in examples/training_scripts/... ?

I am 100% my first attempt was to just create the table object, pass it to the SimulatorBackend, etc, but that failed.

Reply:
Not sure exactly what your setup was, settping dependencies should generally work without passing class name. Regarding creating the object only when first used, this makes sense otherwise the tabular table gets serialized and sent over which may not be what we want.

Reply:
OK, BlackboxRepositoryBackend is a good solution.